aggregator/src/aggregator.rs

```diff
@@ -1825,6 +1835,20 @@ impl VdafOps {
                     // but the leader never got our response and so retried stepping the job.
                     // TODO(issue #1087): measure how often this happens with a Prometheus metric
                     if helper_aggregation_job.round() == leader_aggregation_job.round() {
+                        match helper_aggregation_job.last_continue_request_hash() {
+                            None => {
+                                return Err(datastore::Error::User(Error::Internal(format!(
+                                    "aggregation job {aggregation_job_id} is in round {} but has no last request hash",
```

Comment:
Tiny improvment suggestion: we should check that the continue request does not claim to be for round zero before this, and return some sort of bad request error. This would close off a convoluted route through which an external request could trigger this "internal" error.

Reply:
Good catch. I think we also want some DAP text specifying that an `AggregationJobContinueReq` with `round = 0` is illegal, so I will try to sneak that into DAP-04.